WildAlertApi/WildAlert.UnitTests/UpdateSensorCommandHandlerTests.cs

```diff
@@ -0,0 +1,49 @@
+using FluentAssertions;
+using MapsterMapper;
+using Microsoft.EntityFrameworkCore;
+using WildAlert.Application.Requests.Sensors.Commands.UpdateSensor;
+using WildAlert.Persistence;
+using WildAlert.Persistence.Entities.Sensors;
+using WildAlert.UnitTests.Factories;
+
+namespace WildAlert.UnitTests;
+
+public class UpdateSensorCommandHandlerTests
+{
+    private ApplicationDbContext _context = null!;
+    private IMapper _mapper = null!;
+    
+    [SetUp]
+    public void Setup()
+    {
+        _context = ApplicationDbContextFactory.Create();
+        _mapper = new Mapper();
+
+        _context.Sensors.Add(new SensorEntity {Latitude = 50, Longitude = 10, Name = "not updated sensor"});
+        _context.SaveChanges();
+    }
+
+    [Test]
+    public async Task Handle_ShouldUpdateAndReturnAlert()
```

Comment:
You should also test if it throws an exception if given incorrect/not existing ID